src/lib/homestore_backend/resync_pg_shard.fbs

```diff
@@ -0,0 +1,25 @@
+namespace homeobject;
+
+table Member {
+   uuid : [ubyte];
+   name : [ubyte];
+}
+
+table PGInfoEntry {
+    pg_id : uint32;                     // only low 16 bit is used for pg_id;
+    replica_set_uuid : [ubyte];         // uuid of replica set
+    members : [Member];                 // peers;
+}
+
+table ShardInfoEntry {
+    pg_id : uint32;                     // pg id which this shard belongs to;
```

Comment:
Do we need all these whats there in shard_info_superblk ?
   State state;
    uint64_t created_time;
    uint64_t last_modified_time;
    uint64_t available_capacity_bytes;
    uint64_t total_capacity_bytes;
    uint64_t deleted_capacity_bytes;
    std::optional< peer_id_t > current_leader{std::nullopt};

Reply:
But the create and last modify time and state, we may not get log entries for these. We may have to calculate total bytes for each shard too. It might be easy to use whatever leader gives.

Reply:
it is wrong to sync create/modify time to follower, it should be updated when this receiver member got the message and create the shard on it. Other fields like available/total/deleted, they are not actually used anywhere and even though we want them in the future, they need to be updated incrementally based on this member's put/del blob requests. Syncing from leader will cause wrong result because it is a moving target.